File: Stage4/Java/Problem5597.java
package Stage4.Java;

import java.util.Scanner;

public class Problem5597 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        boolean[] isSubmitted = new boolean[30];

        for (int i = 0; i < 28; i++) {
            int num = sc.nextInt();
            isSubmitted[num - 1] = true;
        }

        for (int i = 0; i < 30; i++) {
            if (!isSubmitted[i]) {
                System.out.println(i + 1);
            }
        }

        sc.close();
    }
}

File: Stage4/C/Problem5597.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    bool isSubmitted[30] = {false};

    for (int i = 0; i < 28; i++) {
        int num;
        scanf("%d", &num);
        isSubmitted[num - 1] = true;
    }

    for (int i = 0; i < 30; i++) {
        if (!isSubmitted[i]) printf("%d\n", i + 1);
    }

    return 0;
}